lib/admin/WP_Auth0_Admin_Appearance.php

```diff
@@ -53,117 +53,91 @@ public function init() {
+				__( 'Available languages list', 'wp-auth0' )
+			)
+		);
 	}
 
-	public function render_custom_js() {
-		$v = $this->options->get( 'custom_js' );
-?>
-      <textarea name="<?php echo $this->options->get_options_name(); ?>[custom_js]" id="wpa0_custom_js"><?php echo esc_attr( $v ); ?></textarea>
-      <div class="subelement">
-        <span class="description"><?php echo __( 'This should be a valid JS to customize the Auth0 login widget to, for example, add custom buttons. ', 'wp-auth0' ); ?><a target="_blank" href="https://auth0.com/docs/hrd#option-3-adding-custom-buttons-to-lock"><?php echo __( 'More info', 'wp-auth0' ); ?></a></span>
-      </div>
-    <?php
+	public function render_primary_color( $args = array() ) {
+		$this->render_text_field( $args[ 'label_for' ], $args[ 'opt_name' ] );
+		$this->render_field_description(
+			__( 'Primary color for the Auth0 login form in hex format. ', 'wp-auth0' ) .
+			$this->get_docs_link(
+				'libraries/lock/v11/configuration#primarycolor-string-',
+				__( 'More information this settings', 'wp-auth0'	)
+			)
+		);
 	}
 
-	public function render_username_style() {
-		$v = $this->options->get( 'username_style' );
-?>
-      <input type="radio" name="<?php echo $this->options->get_options_name(); ?>[username_style]" id="wpa0_username_style_auto" value="" <?php echo esc_attr( $v ) == '' ? 'checked="true"' : ''; ?> />
-      <label for="wpa0_username_style_auto"><?php echo __( 'Auto', 'wp-auth0' ); ?></label>
-
-      <input type="radio" name="<?php echo $this->options->get_options_name(); ?>[username_style]" id="wpa0_username_style_email" value="email" <?php echo esc_attr( $v ) == 'email' ? 'checked="true"' : ''; ?> />
-      <label for="wpa0_username_style_email"><?php echo __( 'Email', 'wp-auth0' ); ?></label>
-
-      <input type="radio" name="<?php echo $this->options->get_options_name(); ?>[username_style]" id="wpa0_username_style_username" value="username" <?php echo esc_attr( $v ) == 'username' ? 'checked="true"' : ''; ?> />
-      <label for="wpa0_username_style_username"><?php echo __( 'Username', 'wp-auth0' ); ?></label>
-
-      <div class="subelement">
-        <span class="description">
-          <?php echo __( 'If you want to allow the user to use either email or password, set it to Auto.', 'wp-auth0' ); ?>
-          <a target="_blank" href="https://auth0.com/docs/libraries/lock/customization#usernamestyle-string-"><?php echo __( 'More info', 'wp-auth0' ); ?></a>
-        </span>
-      </div>
-    <?php
+	public function render_language_dictionary( $args = array() ) {
+		$this->render_textarea_field( $args[ 'label_for' ], $args[ 'opt_name' ] );
+		$this->render_field_description(
+			__( 'The languageDictionary parameter for the Auth0 login form. ', 'wp-auth0' ) .
+			sprintf(
+				'<a href="https://github.com/auth0/lock/blob/master/src/i18n/en.js" target="_blank">%s</a>',
+				__( 'List of all modifiable text', 'wp-auth0' )
```

Comment:
> List of all modifiable text

text ... options? or something that describes that this are items that change how the form is displayed.